recipes/yum_setup.rb

```diff
@@ -0,0 +1,15 @@
+#
+# Cookbook Name:: delugeserver
+# Recipe:: yum_setup
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+include_recipe 'yum'
```

Comment:
to be honest I was staying away from yum-specific strategies to make this cookbook as consumable as possible across multiple platforms...but if this method offers a better overall experience then I think I'll be ok :)